autosklearn/evaluation/util.py

```diff
@@ -42,7 +46,8 @@ def empty_queue(queue_):
     queue_.close()
 
 
-def extract_learning_curve(stack, key=None):
+def extract_learning_curve(stack: List[Dict[str, Any]],
+                           key: Optional[str] = None) -> List[Union[float, int]]:
```

Comment:
How can the return value be an integer?

Reply:
I got confused with MAXINT, reverted this.